projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -286,6 +287,7 @@ if (environment.cpq) {
       provide: USE_MY_ACCOUNT_V2_NOTIFICATION_PREFERENCE,
       useValue: environment.myAccountV2,
     },
+    useAuthorizationCodeFlowByDefaultProvider(false),
```

Comment:
let's not over-provide defailt value explicitly here in the module
the factory `() => false` inside the injection token definition suffices.

now if customer calls `useAuthorizationCodeFlowByDefaultProvider(true)` before importing our module, our module will win - as the last provided, and it might be confusing for customers

for more see https://dev.to/krisplatis/the-importance-of-order-in-angular-dependency-injection-providers-4g31

Reply:
I think there is some confusion here.  The use highlighted here is in the storefrontapp and would be how the customer toggles the feature in their repo.

The token is not provided in any of our modules (and `useAuthorizationCodeFlowByDefaultProvider` is not used either), so the only default from the token's factory would apply normally.  I don't think we can overwrite the customer's specification currently.